test/e2e/_incluster-test-code/memcached_test.go

```diff
@@ -126,7 +126,7 @@ func verifyLeader(t *testing.T, namespace string, f *framework.Framework, labels
 		return true, nil
 	})
 	if err != nil {
-		return nil, fmt.Errorf("error getting leader lock configmap: %v", err)
```

Comment:
@nrchakradhar also nothing in `test/` should be wrapped. Otherwise LGTM.

Reply:
Changes done.